.github/workflows/build_macos.yaml

```
@@ -35,15 +40,15 @@ jobs:
       run: |
         mkdir build_wl
         cd build_wl/
-        ../utils/macos/build_app.sh "--${{ matrix.config }}" "--${{ matrix.compiler }}"
+        ../utils/macos/build_app.sh "--${{ matrix.config }}" "--${{ matrix.compiler }}" "--no-website"
         grep -v REVDETECT-BROKEN ../build/VERSION
         DMGPATH="$(pwd)"
         DMGFILE="$(ls *.dmg)"
         echo "DMG file is: $DMGPATH/$DMGFILE"
-        mv "$DMGFILE" "${{ github.workspace }}/Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.compiler }}-${{ matrix.arch }}.dmg"
+        mv "$DMGFILE" "${{ github.workspace }}/Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.compiler }}-${{ matrix.osname }}.dmg"
```

```suggestion
        mv "$DMGFILE" "${{ github.workspace }}/Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.compiler }}-macos${{ matrix.os }}-${{ matrix.arch }}.dmg"
```

(or:

```suggestion
        ARCH="$(uname -m)"
        DMGNEWNAME="${{ github.workspace }}/Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.compiler }}-macos${{ matrix.os }}-${ARCH}.dmg"
        mv "$DMGFILE" "$DMGNEWNAME"
        echo "arch=$ARCH" >>"$GITHUB_ENV"
        echo "dmg_name=$DMGNEWNAME" >>"$GITHUB_ENV"
```
and use
```
path: ${{ env.dmg_name }}
```
in the upload step to remove the duplication)